lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -528,10 +557,63 @@ public void setProgress(@FloatRange(from = 0f, to = 1f) float progress) {
     }
   }
 
+  /**
+   * @see #setLoopMode(int)
+   */
+  @Deprecated
   public void loop(boolean loop) {
     animator.setRepeatCount(loop ? ValueAnimator.INFINITE : 0);
   }
 
+  /**
+   * Defines what this animation should do when it reaches the end. This
+   * setting is applied only when loop is enabled
+   * Defaults to {@link #NONE}.
+   *
+   * @param loopMode {@link #NONE}, {@link #RESTART} or {@link #REVERSE}
+   */
+  public void setLoopMode(@LoopMode int loopMode) {
+    if (loopMode == NONE) {
+      animator.setRepeatCount(0);
+    } else {
+      if (animator.getRepeatCount() == 0) animator.setRepeatCount(ValueAnimator.INFINITE);
```

Comment:
@fabionuno Why do you have to modify repeat count here? Setting the mode should be sufficient. In other words, it can be NONE with a non-zero repeat count. 
This should mirror the [behavior of ValueAnimator](https://android.googlesource.com/platform/frameworks/base/+/refs/heads/master/core/java/android/animation/ValueAnimator.java#879) as much as possible.

Also, single statement ifs should still have braces.

Reply:
@gpel To start a loop with ValueAnimator we should use `setRepeatCount` and `setRepeatMode`. And to disable the loop we should call `setRepeatCount` with zero.

But I think most people use simple loops with animation restarting infinitely.
I've tried to create an easier way, just setting the `setLoopMode(Restart)` without the need to explicitly use `setLoopCount(Infinite)`.

We can make Lottie works exactly like `ValueAnimator` or we can keep the current proposal, saving the  LoopCount value when LoopMode is set to None and restore it if `LoopMode` is changed.

What do you think is the best way?

Reply:
@fabionuno I think this should mirror ValueAnimator as closely as possible meaning they can basically be pass through APIs to LottieValueAnimator itself. Since we added repeatCount as part of this PR, we can remove NONE and have the APIs be properly straight passed through to ValueAnimator. In that case, it would make sense to rename the methods from loop to repeat since REVERSE isn't really a loop technically